frontend/src/components/Home.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import "../App.css";
+import { motion } from "framer-motion";
+
+const ComingSoon: React.FC = () => {
+  const navigate = useNavigate();
+
+  return (
+    <div className="coming-soon-container">
```

Comment:
```suggestion
    <section className="coming-soon-container">
```
Whenever possible, let's try to use semantic HTML tags, even if just as practice